web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -46,17 +46,15 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
     },
 
     _addClearButton: function() {
-
-        this.add(
-            new Ext.Button({
-                cls: "clearFiltersButton",
-                text: OpenLayers.i18n('clearFilterButtonLabel'),
-                listeners: {
-                    scope: this,
-                    click: this.resetConstraints
-                }
-            })
-        );
+        this.resetLink = new Ext.ux.Hyperlink({
+            anchorCls: 'resetText',
+            cls: "clearFiltersLink",
+            text: OpenLayers.i18n('clearLinkLabel', {text: 'Clear and Reset'})
```

Comment:
This is changing the text of the button from whatever is in `clearFilterButtonLabel` to 'Clear and Reset'. Is that deliberate or a mistake?